app/services/authenticationService.js

```diff
@@ -9,7 +9,7 @@ function authenticationService(){
 
 	function logIn(user){
 		var $q = $.Deferred();		
-		$.when($.post( "/api/sessions",{ username: user.username, password: user.password}))
+		$.post( "/api/sessions",{ username: user.username, password: user.password})
```

Comment:
You can avoid create a promise. Just return $.post which already return a promise.
`return $.post("/api/sessions", ...`